### PR TITLE
Section 2 suggestions

### PR DESCRIPTION
**Pull request by andyfaff**

### CODE REVIEW

File: src/tex/ms.tex [hunk 1]
```diff
@@ -120,7 +120,8 @@ \section{Introduction}
 \section{Prior}
 \label{sec:prior}
 
-The most common prior probability that is used for a parameter is uniform between two values.
+% ARJN Application of a half-closed distribution may be as common? e.g. [0, inf] for a parameter that is known to be non-negative. Should we mention this in table 1?
```
Comment (arm61) [inline src/tex/ms.tex:123]: It's a half closed distribution possible in a Bayesian sampler? Would this not lead to an 0 probability given that $p(U) = \frac{1}{b-a}$, where one of $a$ or $b$ would be infinity?

File: src/tex/ms.tex [hunk 2]
```diff
@@ -166,8 +169,9 @@ \section{Prior}
 \end{figure}
 %
 
-In the case that the prior distribution cannot be described easily with some mathematical function, for example, if it is a multimodal model result from some other sampling method, then the chain from the prior sampling should be given. 
-The chain is all of the samples investigated in the sampling and should be shared, although in the case that this chain is very large a subsampled object may be shared, in which case, the autocorrelation analysis performed should be described (see Section~\ref{sec:posterior} for a more complete discussion of this).
+It is possible that the prior distribution cannot be described with a simple mathematical function, with multi-modal priors being an example.
```
Comment (arm61) [inline src/tex/ms.tex:172]: Not all multi-model priors cannot be described with a mathematical function though.

**Review by arm61 — COMMENTED** [review]
I am happy to specific that the "recommendation" is the concise version but it is not bad to include the full equation.

File: src/tex/ms.tex (hunk 1 above)
Comment (andyfaff) [reply src/tex/ms.tex:123]: With emcee and pymc (v3/4) I know that half-closed distributions are possible for priors. In the case of emcee you would do:

```
def log_prior(theta):
    # parameter specified in [0, inf]
    if theta < 0:
        return -inf
    return 0
```

i.e. probability is not "scaled" correctly, in that the log-prior has no numerical effect if the parameter is in a possible region. However, if the parameter is in an infeasible region the log-prior makes the probability -np.inf (impossible).

Comment (arm61) [reply src/tex/ms.tex:123]: Oh yeah, that is a bit daft of me cause I wrote code that does exactly that earlier...It is probably worth a passing mention then.

**Pull request merged**